src/components/tooltip/Tooltip.tsx

```diff
@@ -332,17 +332,17 @@ class Tooltip extends React.Component<TooltipProps, State> {
                 onContextMenu={this.handleTooltipEvent}
                 onKeyPress={this.handleTooltipEvent}
             >
-                <div role="tooltip" aria-live="polite" aria-hidden={isLabelMatchingTooltipText}>
+                <div aria-live="polite" aria-hidden={isLabelMatchingTooltipText}>
                     {tooltipInner}
                 </div>
             </div>
         ) : (
             <div
                 className={classes}
                 id={this.tooltipID}
-                role="tooltip"
                 aria-live="polite"
                 aria-hidden={isLabelMatchingTooltipText}
+                {...(theme !== TooltipTheme.ERROR ? { role: 'tooltip' } : {})}
```

Comment:
Can we pull this into a variable so it can be reused and consistent on line 335? Separately, does `role={theme === TooltipTheme.ERROR ? undefined : 'tooltip'}` work? We generally prefer positive rather than negative conditions unless it's a guard clause.

Reply:
@jstoffan I just updated this PR with your comments